### PR TITLE
Implements Model Admin to register models

### PR DESCRIPTION
**Pull request by Hamza-Chaudhary65**
- Implements Model Admin to register models

### CODE REVIEW

**Review by ZAINABFATIMA0 — APPROVED** [review]

**Review by sheralim012 — CHANGES_REQUESTED** [review]
https://django-extensions.readthedocs.io/en/latest/admin_generator.html

**Review by ZAINABFATIMA0 — APPROVED** [review]

File: fintrack/users/admin.py [hunk 1]
```diff
@@ -2,4 +2,28 @@
 
 from . models import User
 
-admin.site.register(User)
+
+@admin.register(User)
+class UserAdmin(admin.ModelAdmin):
+    list_display = (
+        'id',
+        'password',
```
Comment (sheralim012) [inline fintrack/users/admin.py]: do we need to show password?
don't just blindly follow things

File: fintrack/fintrack/settings.py [hunk 2]
```diff
@@ -45,7 +45,7 @@
     'users',
     'rest_framework',
     'rest_framework_simplejwt',
-    
+    'django_extensions'
```
Comment (sheralim012) [inline fintrack/fintrack/settings.py:48]: update requirements.txt file

File: fintrack/banks/admin.py [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+@admin.register(Bank)
+class BankAdmin(admin.ModelAdmin):
+    list_display = ('id', 'name')
+    search_fields = ('name',)
+
+
+@admin.register(Branch)
+class BranchAdmin(admin.ModelAdmin):
+    list_display = ('id', 'name', 'code', 'address', 'phone_number', 'bank')
+    list_filter = ('bank',)
```
Comment (sheralim012) [inline fintrack/banks/admin.py]: it's not recommended to include foreign keys into filter
reason is, when there is large amount of different objects in foreign key listing, it breaks the page

**Review by sheralim012 — CHANGES_REQUESTED** [review]

**Review by sheralim012 — APPROVED** [review]

**Pull request merged**